examples/c/CMakeLists.txt

```diff
@@ -9,10 +9,13 @@ project( regoc VERSION 0.1.0 LANGUAGES C CXX )
 
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 
+set(REGOCPP_C_WRAPPER_REPO "https://github.com/microsoft/rego-cpp/" CACHE STRING "Specifies the git repository URL to test the C-Wrapper")
```

Comment:
so per my comment above, if we set the default values of these from an environment variable instead of in CMake, then I think we can use it for all such cases (all of the wrappers will have this same issue)